doc/modules/multiclass.rst

```diff
@@ -30,21 +30,21 @@ by decomposing such problems into binary classification problems.
 
   - **Multioutput-multiclass classification** and **multi-task classification**
     means that a single estimator has to handle
-    several joint classification tasks. This is a generalization
+    several joint classification tasks. This is both a generalization
     of the multi-label classification task, where the set of classification
-    problem is restricted to binary classification, and of the multi-class
+    problems is restricted to binary classification, as well as a generalization of the multi-class
     classification task. *The output format is a 2d numpy array or sparse
     matrix.*
 
     The set of labels can be different for each output variable.
-    For instance a sample could be assigned "pear" for an output variable that
-    takes possible values in a finite set of species such as "pear", "apple",
-    "orange" and "green" for a second output variable that takes possible values
+    For instance, a sample could be assigned "pear" for an output variable that
+    takes possible values in a finite set of species such as "pear" and "apple", 
+    and "orange" or "green" for a second output variable that takes possible values
```

Comment:
I think you changed the meaning. It was a list where it should be "or" in the end, not "and".